code42.json

```diff
@@ -1065,6 +1065,243 @@
+                        "org_id"
+                    ],
+                    "default": "",
+                    "order": 0
+                },
+                "username": {
+                    "description":  "Username to create",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "email"
+                    ],
+                    "default": "",
+                    "order": 1
+                },
+                "password": {
+                    "description":  "Password for newly created user",
+                    "verbose": "Defaults to no password set, which leaves user in 'invited' state. They will need to reset password via email prior to first login.",
+                    "data_type": "password",
+                    "required": false,
+                    "primary": false,
+                    "default": null,
+                    "order": 3
+                },
+                "first_name": {
+                    "description":  "User's first name",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "default": "",
+                    "order": 4
+                },
+                "last_name": {
+                    "description":  "User's last name",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "default": "",
+                    "order": 5
+                },
+                "notes": {
+                    "description":  "Notes about user",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "default": "",
+                    "order": 6
+                }
+
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.username",
+                    "data_type": "string",
+                    "contains": [
+                        "email"
+                    ],
+                    "column_name": "Username",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.data.userUid",
+                    "data_type": "string",
+                    "column_name": "User ID",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "status",
+                    "column_order": 2
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
```

Comment:
The summary is more accessible if you reference the props you add to it via dot notation,

like
```
 "data_path": "action_result.summary.user_id"
```

etc, and then you can remove this bare summary one